extraction/main.py

```diff
@@ -0,0 +1,13 @@
+"""This script starts the program"""
+
+import extractor
+
+# URL = 'https://theoatmeal.com/'
+URL = 'http://www.whatshot.in/chandigarh/' \
+      'social-distancing-5-amazing-web-series-to-binge-watch-right-now-c-20716'
+# URL = 'https://www.html5rocks.com/en/tutorials/video/basics/'
```

Comment:
Can you pass URL as args?
https://docs.python.org/3.3/library/argparse.html

Reply:
Added!